src/lib/components/01-elements/icon/icon.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+interface IconProps {
+  name: string;
+  className?: string;
+}
+
+const Icon: React.FC<IconProps> = ({ name, ...props }) => {
+  return (
+    <svg {...props} aria-hidden="true" >
+      <use href={`/dist/sprite.svg#${name}`}></use>
```

Comment:
This probably won't work when imported in a Next site. It would probably be better to have a way to import independent svg files using webpack and not a sprite. Please check other next implementation we've done in the past in Manatí.